### ISSUE TITLE
deleting hunts very very very slow in 0.72.0/0.72.1

### ISSUE THREAD

**Issue by awx-vsyr** [Velocidex/velociraptor#3485]
Hello, 

we have a cleanup task for housekeeping server hunts. [unfortunately at the moment for self inflicted reasons client monitoring tasks don't work for us]
with  0.72.0/1 it seems like it can't even do 1 or max 1-2 hunts in 10 minutes. (I've added logging to the server event artifact at Mike's request to quantify)

Seemingly the cpu/io/memory perf on the server remained the same. The server is not heavily loaded and there's basically no other hunts happening most of the time . [although I should really check the g nfs performance - but across the master minion no obvious issues  ]

**Comment by mcohen-r7** [comment]
This likely happens because IO is very slow - you can make it go faster by parallelizing the operations

```sql
SELECT * FROM foreach(row={
  SELECT * FROM hunts()
}, query={
  SELECT * FROM hunt_delete(hunt_id=hunt_id, really_do_it=TRUE)
}, workers=10)
```

We can probably still speed things up a bit more but the above approach will help

**Comment by scudette** [comment]
Please test with CI build after #3560 and #3561 which should improve performance a lot.

**Comment by scudette** [comment]
Fixed in 0.72.4
